check_kernel

```diff
@@ -137,7 +148,11 @@ def installed_kernel_versions_fedora():
 
 
 def installed_kernel_version():
-    return sorted(installed_kernel_versions())[-1]
+    installed = sorted(installed_kernel_versions())
+    if not installed:
+        print('KERNEL UNKNOWN - unable to determine running kernel version')
+        sys.exit(3)
+    return installed[-1]
```

Comment:
Before the fix, we have been presented with a Python Traceback, because the list of determined packages was empty.

Reply:
lgtm